docs/source/architecture/poet.rst

```diff
@@ -590,7 +590,7 @@ Comments on Multi-user or Multi-ledger SGX Enclave Service
 ----------------------------------------------------------
 
 It is possible to use the same enclave for multiple users or ledgers by making
-username and ledgername input **Parameters** to ``generateSignUpData()`` and
+username and ledgername, as input parameters to ``generateSignUpData()`` and
```

Comment:
I agree that this sentence was confusing, and I'm glad you're improving it.  However, this change is grammatically incorrect ("as" doesn't work with "make").  

I suggest saying "... by using username and ledgername as input parameters...". Note that there should not be a comma before "as".

P.S. I'm OK with "use" and "using" in the same sentence, but if you don't like it, you could say "by providing username and ledger name as input parameters".